lua/compe_path/init.lua

```diff
@@ -2,7 +2,7 @@ local compe = require'compe'
 
 -- TODO: ' or " or ` is valid as filename
 local NAME_PATTERN = [[\%([^/\\:\*?<>'"`\|]\)]]
-local DIRNAME_REGEX = vim.regex(([[\%(/PAT\+\)*\ze/PAT*$]]):gsub('PAT', NAME_PATTERN))
+local DIRNAME_REGEX = vim.regex(([[\%(\$PAT\+\)\?\%(/PAT\+\)*\ze/PAT*$]]):gsub('PAT', NAME_PATTERN))
```

Comment:
We shouldn't modify it.

Sorry for the confusing, but the current implementation is handle `$HOME/.config/nvim/init.vim` as `{ prefix = '$HOME/', dirname = '.config/nvim/init.vim' }`

So we don't have to include `$HOME` as dirname.

Reply:
I see, I added what you sent in your other comment.